src/homepageExperience/components/steps/arduino/InstallDependencies.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  ComponentColor,
+  Button,
+  ButtonGroup,
+  Orientation,
+} from '@influxdata/clockface'
+
+import React, {FC, useEffect, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {getBuckets} from 'src/buckets/actions/thunks'
+import {event} from 'src/cloud/utils/reporting'
+import {keyboardCopyTriggered, userSelection} from 'src/utils/crossPlatform'
+
+export const InstallDependencies: FC = () => {
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(getBuckets())
+  }, [])
+
+  const headingWithMargin = {marginTop: '48px', marginBottom: '0px'}
+
+  const logCopyCodeSnippetMac = () => {
+    event('firstMile.arduinoWizard.installDependenciesMac.code.copied')
+  }
+  const logCopyCodeSnippetWindows = () => {
+    event('firstMile.arduinoWizard.installDependenciesWindows.code.copied')
+  }
+  const logCopyCodeSnippetLinux = () => {
+    event('firstMile.arduinoWizard.installDependenciesLinux.code.copied')
+  }
+
+  type CurrentOSSelection = 'Linux' | 'Mac' | 'Windows'
+  const [currentSelection, setCurrentSelection] = useState<CurrentOSSelection>(
+    'Mac'
+  )
+
+  useEffect(() => {
+    const fireKeyboardCopyEvent = event => {
+      if (
+        keyboardCopyTriggered(event) &&
+        userSelection().includes('brew install')
+      ) {
+        logCopyCodeSnippetMac()
+      }
+      if (
+        keyboardCopyTriggered(event) &&
+        (userSelection().includes('Expand-Archive') ||
+          userSelection().includes('mv'))
+      ) {
+        logCopyCodeSnippetWindows()
+      }
+      if (
+        keyboardCopyTriggered(event) &&
+        (userSelection().includes('wget') ||
+          userSelection().includes('tar') ||
+          userSelection().includes('sudo'))
+      ) {
+        logCopyCodeSnippetLinux()
+      }
+    }
+    document.addEventListener('keydown', fireKeyboardCopyEvent)
+    return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
+  }, [])
+
+  return (
+    <>
```

Comment:
The fragment (`<>`) is needed here, becuase if it wasn't here, this would return a component tree with no root component - `<h1>Install dependencies</h1>` and `<ButtonGroup>` are siblings, and React components need to render with a single root component.